css/main.css

```diff
@@ -8,3 +8,134 @@ body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
 }
+
+section {
+  margin: 200px 100px 0 100px;
+}
+#phones {
+
+}
+h1 {
+  margin-top: -20px;
+  margin: 0px 0;
+ color: #06365f;
+ font-size: 1.5em;
+}
+#brand {
+  margin-left: -50px;
+  margin-right: 100px;
+  margin-top: 20px;
+  width: 300px;
+  height: 75px;
+}
+article {
+  background-color: #1c5380;
+}
+#loginbutton {
+
+   box-shadow: -1px 1px 5px black;
+   background: rgb(28,83,128);
+   background: linear-gradient(0deg, rgba(28,83,128,1) 0%, rgba(77,131,176,1) 70%);
+
+ color: white;
+ font-size: 1em;
+ border: none;
+ margin: 35px 80px 70px 1px;
+ height: 40px;
+ width: 100px;
+ border-radius: 2px;
+}
+button img {
+width: 25px;
+}
+article {
+  
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin: 0 60px;
+
+}
+body section {
+  display: flex;
+}
+
+#text {
+  background-color: white;
+  background-repeat: no-repeat;
+  border: 2px solid white;
+  border-radius: 10px;
+  padding: 40px;
+  margin: 0 80px;
+  font-size: 16px;
+  box-shadow: 0px -3px 3px rgb(43, 42, 42);
+}
+#buttons {
+  display: flex;
+  justify-content: flex-start;
+}
+#apple {
+  background-color: transparent;
+  border: none;
+  background-image: url(/lab-css-instagram-clone/images/badge-iphone.png);
```

Comment:
Does the url works for you? for me I needed to do it with (/images/badge-iphone.png)
And idem with all the others in html and css